<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Skillinspector</class>
 <widget class="QGroupBox" name="Skillinspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>Skill Inspector</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="SkillName">
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Skill Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
